core/templates/dev/head/domain/objects/FractionObjectFactorySpec.js

```diff
@@ -64,7 +64,7 @@
      it('should throw errors for invalid fractions', function() {
        // Invalid characters.
        expect(function() {Fraction.fromRawInputString('3 \ b')}).toThrow(
-         new Error(errors.INVALID_CHARS));
+          new Error(errors.INVALID_CHARS));
```

Comment:
Deindent by 1? (And two lines down, deindent by 2)

Reply:
Done.